src/Groups/GAPGroups.jl

```diff
@@ -261,8 +261,57 @@ Return the identity of the parent group of `x`.
 Base.one(x::GAPGroupElem) = one(parent(x))
 
 Base.show(io::IO, x::GAPGroupElem) = print(io, String(GAPWrap.StringViewObj(x.X)))
+
+
+# Printing GAP groups
 Base.show(io::IO, x::GAPGroup) = print(io, String(GAPWrap.StringViewObj(x.X)))
```

Comment:
This generic method, i. e., delegate to `GAPWrap.StringViewObj` is currently used by f.p. groups. It could be replaced by
```
function Base.show(io::IO, x::PermGroup)
  @show_name(io, x)
  @show_special(io, x)
  print(io, "Group")
  if !get(io, :supercompact, false)
    if has_order(x)
      print(io, " of order ", order(x))
    end
  end
end
```

Reply:
OK